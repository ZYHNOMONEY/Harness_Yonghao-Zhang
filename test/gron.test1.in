{
    "company": "Tech Corp",
    "employees": [
        {"name": "John Doe Yonghao", "department": "Engineering", "years": 5},
        {"name": "Jane Smith Zhang", "department": "Marketing", "years": 3}
    ],
    "location": {
        "city": "New York",
        "address": "123 Tech Street",
        "zip": "10001"
    },
    "revenue": 5000000,
    "publicly_traded": false
}
