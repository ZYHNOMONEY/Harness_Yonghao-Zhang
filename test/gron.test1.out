o.company = "Tech Corp"
o.employees.0.name = "John Doe Yonghao"
o.employees.0.department = "Engineering"
o.employees.0.years = 5
o.employees.1.name = "Jane Smith Zhang"
o.employees.1.department = "Marketing"
o.employees.1.years = 3
o.location.city = "New York"
o.location.address = "123 Tech Street"
o.location.zip = "10001"
o.revenue = 5000000
o.publicly_traded = false